pkg/sentry/socket/netlink/route/protocol.go

```diff
@@ -236,6 +356,10 @@ func (p *Protocol) ProcessMessage(ctx context.Context, hdr linux.NetlinkMessageH
 		}
 	}
 
+	if hdr.Type == linux.RTM_GETROUTE && hdr.Flags == linux.NLM_F_REQUEST {
```

Comment:
I kind of feel like this and the next check should be done in each dump method and just update their doc to say that they handle RTM_GET(LINK|ADDR|ROUTE) requests.

Reply:
I've tried to fix like you suggest. Please check if the new version is what you mean. Thanks!